libs/features/pages/consultancy-page/src/lib/consultancy-hero-section/consultancy-hero-section.component.scss

```diff
@@ -0,0 +1,89 @@
+/* Layout Properties for larger screens */
+.consultancy-main {
+    top: 0px;
+    left: 0px;
+    width: 1920px;
+    height: 1082px;
+    /**UI Properties*/
+    background: #dbc2f5 0% 0% no-repeat padding-box;
```

Comment:
For reuse of colors, use CSS variables specified in the elements/theming/_variables.scss file to specify colors/background colors